src/views/officer-table.j2

```diff
@@ -0,0 +1,29 @@
+<table>
+	<thead>
+		<tr>
+			{% for field in metadata.fields %}
+				<th>{{ field.Label }}</th>
+			{% endfor %}
+		</tr>
+	</thead>
+
+	<tbody>
+		{% for record in records %}
+			{# records are ordered by date, starting with the present into the past, so the "previous" record will be the next item in the list of records #}
+			{% set previous_record = loop.nextitem %}
+			<tr>
+				{% for field in metadata.fields %}
+					{% set classname = record | diff_classname(previous_record, field) %}
+					<td class="{{ classname }}">
+						{% set value = record.get(field.FieldName) | string %}
+						{% if value and value.startswith("http") %}
+							<a href="{{ value }}" target="_blank" rel="noopener noreferrer">{{ value }}</a>
+						{% else %}
+							{{ value if value }}
```

Comment:
Since you're casting this to `string` on line 18, can this just be `{{ value}}`?

Reply:
I'm not sure! I copied that part from `officer.j2` 😅 Presumably it would just be an empty string if falsy, right?

Reply:
Removed this check in feb04b2

Reply:
Oh jeez 😂 it can probably be changed there too. And yea! Should be an empty string at this point